### PR TITLE
Added URLs to link text

### PR DESCRIPTION
**Pull request by shawnthompson**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -23,46 +23,46 @@
...
+- [Accessibility Meetup Survey](http://www.dennisdeacon.com/a11y-meetup-survey-2018/)
+- [A11yTalks](https://a11ytalks.com/)
 
 ## Government Resources
-- 18F's Tools List - https://accessibility.18f.gov/tools/
-- French Government Resources - Ressources RGAA - Collated by Access42 - http://gta2017.access42.net/2/ressources/
-- List Government Resources & Good Links from Lainey Feingold http://www.lflegal.com/2013/05/gaad-legal/
+- [18F's Tools List](https://accessibility.18f.gov/tools/)
+- [French Government Resources - Ressources RGAA - Collated by Access42](http://gta2017.access42.net/2/ressources/)
+- [List Government Resources & Good Links from Lainey Feingold](http://www.lflegal.com/2013/05/gaad-legal/)
```
Comment (weboverhauls) [inline README.md]: Thanks for doing this!
I'm getting an error on Lainey's article; the URL may have changed to https://www.lflegal.com/2013/05/gaad-2013/
Please update PR and then I'll merge.

Comment (shawnthompson) [reply README.md]: I wonder if the link should be this page: https://www.lflegal.com/resources/

Comment (weboverhauls) [reply README.md]: Sure!

Comment (shawnthompson) [reply README.md]: Thanks @weboverhauls. I think this list needs to be revised. A few other broken links so lets merge this and I'll new PR to fix them.

**Pull request merged**